client/lib/main.js

```diff
@@ -56,7 +56,35 @@ function main(options, callbacks) {
     // TODO: Get URI from mappings.
     uri = "http://localhost:8080/bundles/main.js";
   }
-  SANDBOX(uri, function(sandbox) {
+  SANDBOX(uri, {
+    global: {
+      // Not available in this context.
+      Components: undefined
+    },
+    onInitModule: function(moduleInterface, moduleObj, pkg, sandbox, options) {
+      var origRequire = moduleObj.require;
+      moduleObj.require = function(identifier) {
+        if (
+          identifier === "chrome" ||
+          /^sdk/.test(identifier)
+        ) {
+          // Return SDK module that is not bundled.
+          return require(identifier);
```

Comment:
Override `require()` within the bundle (which includes statically referenced firebug.sdk modules) to intercept calls to SDK modules that are not included in the bundle.